### ISSUE TITLE
Cannot disabled `import/no-unresolved` by inline comment

### ISSUE THREAD

**Issue by huangyingjie** [import-js/eslint-plugin-import#728]
````js
    /* eslint import/no-unresolved: 0 */
    // webpack.base.config.js
    new webpack.DllReferencePlugin({
      context: __dirname,
      manifest: require('./public/dist/vendor-manifest-not-exists-yet.json')
    })
````
This doesnot work. It always report error.
Here is my .eslintrc.js
````js
  'settings': {
    'import/resolver': {
      'webpack': {
        'config': 'webpack.base.config.js'
      }
    }
  },
````
## version
"eslint": "3.12.2",
"eslint-plugin-import": "^1.16.0",
"eslint-import-resolver-webpack": "^0.6.0",

**Comment by ljharb** [comment]
What line is reporting an error, and specifically which error?

**Comment by soryy708** [comment]
Looks like an issue specific to OPs configuration,
they were asked a follow-up question, and we didn't get an answer for 7 years, so I assume they figured it out since.
